ethereum/chain.py

```diff
@@ -59,11 +60,11 @@ def __init__(self, genesis=None, env=None, coinbase=b'\x00' * 20, \
 
         self.head_hash = self.state.prev_headers[0].hash
         self.genesis = Block(self.state.prev_headers[0], [], [])
-        self.db.put('state:'+self.head_hash, self.state.trie.root_hash)
+        self.db.put('state:'.encode('utf8') + self.head_hash, self.state.trie.root_hash)
```

Comment:
I believe that `self.db.put(b'state:' + self.head_hash, self.state.trie.root_hash)` accomplishes the same thing here with better readability.